api-reference/_hidden/PdfCell/wordWrapEnabled.md

```diff
@@ -4,7 +4,8 @@ type: Boolean
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to enable word wrapping for exported data in the resulting PDF file.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+<!-- Description goes here -->
+The default value depends on the value of the grid's [wordWrapEnabled](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/wordWrapEnabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#wordWrapEnabled') option.
```

Comment:
The default value depends on the value of the **DataGrid** [wordWrapEnabled](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/wordWrapEnabled.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#wordWrapEnabled') option.